pkg/graphql/execution_engine_v2_test.go

```diff
@@ -788,6 +799,67 @@ func TestExecutionEngineV2_Execute(t *testing.T) {
 		},
 	))
 
+	schemaWithCustomScalar, _ := NewSchemaFromString(string(`
+    scalar Long
+    type Asset {
+      id: Long!
+    }
+    type Query {
+      asset: Asset
+    }
+  `))
+	t.Run("query with custom scalar", runWithoutError(
+		ExecutionEngineV2TestCase{
+			schema: schemaWithCustomScalar,
+			operation: func(t *testing.T) Request {
+				request := Request{}
+				body := execution.GraphqlRequest{
+					Query:         `{asset{id}}`,
+					OperationName: "",
+					Variables:     nil,
+				}
+				jsonBytes, _ := json.Marshal(body)
+				UnmarshalRequest(bytes.NewBuffer(jsonBytes), &request)
```

Comment:
```suggestion
				_ := UnmarshalRequest(bytes.NewBuffer(jsonBytes), &request)
```

Reply:
Ups, sorry. I have meanwhile added a check on the return value.